src/__test__/test-utils/mockAPI.js

```diff
@@ -44,6 +47,15 @@ const generateDefaultMockAPIResponses = (experimentId) => ({
   [`experiments/${experimentId}/backendStatus`]: () => promiseResponse(
     JSON.stringify(backendStatusData),
   ),
+  '/experiments': () => promiseResponse(
+    JSON.stringify(experimentsData),
+  ),
+  '/projects': () => promiseResponse(
+    JSON.stringify(projectsData),
+  ),
+  'projects/380c53b4-mock-test-8091-4c82336d6d49/samples': () => promiseResponse(
```

Comment:
Do you think you could make it `projects/${projectId}/samples` as you did with `experiments/${experimentId}/backendStatus`

Reply:
I can for this one, but it's not going to be very sustainable. We have to have a way to generate mock data soon.